public/app/features/plugins/admin/components/PluginUsage.tsx

```diff
@@ -0,0 +1,90 @@
+import { css } from '@emotion/css';
+import React, { useMemo } from 'react';
+import { useAsync } from 'react-use';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import { of } from 'rxjs';
+
+import { GrafanaTheme2, PluginMeta } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { Alert, Spinner, useStyles2 } from '@grafana/ui';
+import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
+import { SearchResultsTable } from 'app/features/search/page/components/SearchResultsTable';
+import { getGrafanaSearcher, SearchQuery } from 'app/features/search/service';
+
+type Props = {
+  plugin: PluginMeta;
+};
+
+export function PluginUsage({ plugin }: Props) {
+  const styles = useStyles2(getStyles);
+
+  const searchQuery = useMemo(() => {
+    return {
+      query: '*',
+      panel_type: plugin.id,
+      kind: ['panel'],
+    } as SearchQuery;
+  }, [plugin]);
+
+  const results = useAsync(() => {
+    return getGrafanaSearcher().search(searchQuery);
+  }, [searchQuery]);
+
+  const found = results.value;
+  if (found?.totalRows) {
+    return (
+      <div className={styles.wrap}>
+        <div className={styles.info}>
+          {plugin.name} is used <b>{found.totalRows}</b> times.
+        </div>
+        <AutoSizer>
+          {({ width, height }) => {
+            return (
+              <SearchResultsTable
+                response={found}
+                width={width}
+                height={height}
+                clearSelection={() => {}}
+                keyboardEvents={of()}
+                onTagSelected={() => {}}
```

Comment:
Really nice that we can reuse this `SearchResultsTable` in different areas. Not a huge fan of having to include dummy props though. As this component matures we will probably want to do some thinking on how we structure its interface

On a related note, it seems like we are including the `Tags` column in the search results table, but it is never being populated (as far as I can tell) so if we can hide this column (as part of props or something) when it's not used that might be nice as well

Reply:
Yes -- would love to make the search easier to embed in general, but I think this is an OK first step.

I added some magic to hide the tags field when we know we do not have them